app/controllers/api/v1/run_users_controller.rb

```diff
@@ -12,7 +12,7 @@ def destroy
   end
 
   def update
-    render json: UserSerializer.new(User.update(params[:user_id], user_params))
+    render json: UserSerializer.new(Run.find(params[:run_id]).users.update(params[:user_id], user_params))
```

Comment:
Had to change the update to work with addition of username and passwords for users.